sirepo/package_data/static/js/srw.js

```diff
@@ -2533,6 +2534,20 @@ SIREPO.app.directive('simulationStatusPanel', function(appState, beamlineService
                 return $scope.model == 'coherentModesAnimation';
             }
 
+            function checkRunMode(runMode) {
+                return (runMode == 'sequential') || (runMode == 'parallel');
+            }
+
+            function coherentModesEnoughCores() {
+                var a = appState.models.coherentModesAnimation;
+                if (checkRunMode(a.jobRunMode)) {
+                    return true;
+                }
+                if (a.sbatchCores < coreMinimum) {
+                    throw new Error(`${a.sbatchCores} cores found, should be > 3 for coherent modes`);
```

Comment:
Does this show nicely to the user?

Reply:
Do you want the error coming through in the sim status panel?

Reply:
Yes, because the user should know. On production, they won't see an error if parallel is set, but will see an error if sequential is set. @moellep please correct me if I'm wrong about sequential.

Reply:
I think it is OK if the error shows up in the page header - it won't be common except on dev. Single-core execution isn't supported for srw - see #4049

Reply:
Wrong issue number - the correct is #3886 SRW can't run the coherent modes in serial mode on prod anyway.

Reply:
I agree. Let's make this server side and throw a UserAlert.